File: Source/Addins/MonoDevelop.GtkSharp.Addin/Templates/File/Data/Widget.fs
namespace ${Namespace}

open Gtk

type ${EscapedIdentifier} (builder : Builder) =
    inherit Box(builder.GetObject("${EscapedIdentifier}").Handle)

    new() = new ${EscapedIdentifier}(new Builder("${Namespace}.${EscapedIdentifier}.glade"))

File: Source/Addins/MonoDevelop.GtkSharp.Addin/Templates/File/Data/Widget.cs
using System;
using Gtk;
using UI = Gtk.Builder.ObjectAttribute;

namespace ${Namespace}
{
    public class ${EscapedIdentifier} : Box
    {
        public ${EscapedIdentifier}() : this(new Builder("${Namespace}.${EscapedIdentifier}.glade")) { }

        private ${EscapedIdentifier}(Builder builder) : base(builder.GetObject("${EscapedIdentifier}").Handle)
        {
            builder.Autoconnect(this);
        }
    }
}
